grid/social-media/posts/maps/linkedin.test.ts

```diff
@@ -0,0 +1,47 @@
+import { SuperfaceTest } from '@superfaceai/testing';
+
+const provider = 'linkedin';
+// DevTestCo - https://docs.microsoft.com/en-us/linkedin/marketing/integrations/community-management/organizations#test-organizations
+const profileId = 'urn:li:organization:2414183';
+
+describe(`social-media/posts/${provider}`, () => {
+  let superfacePosts: SuperfaceTest;
+
+  beforeEach(() => {
+    superfacePosts = new SuperfaceTest({
+      profile: 'social-media/posts',
+      provider,
+    });
+  });
+
+  describe('GetProfilePosts', () => {
+    describe('when getting latest posts', () => {
+      it('succeeds', async () => {
+        await expect(
+          superfacePosts.run({
+            useCase: 'GetProfilePosts',
+            input: {
+              profileId,
+            },
+          })
+        ).resolves.toMatchSnapshot();
+      });
+
+      it('correctly paginates', async () => {
+        for (let i = 0, page = null; i < 2; i++) {
```

Comment:
I found an error, when I try to paginate to third page the test starts to fail, because the page link query parameters returned by LinkedIn are uri encoded twice.

<img width="614" alt="image" src="https://user-images.githubusercontent.com/5206165/155373337-b0c5cbc1-8b49-4c7f-8d0c-1df9e8848f07.png">

```
        for (let i = 0, page = null; i < 3; i++) {
```

Reply:
Good catch, I will check it.